libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2772,7 +2773,7 @@ void GCS_MAVLINK::handle_data_packet(mavlink_message_t *msg)
 
 void GCS_MAVLINK::handle_vision_position_delta(mavlink_message_t *msg)
 {
-    AP_VisualOdom *visual_odom = get_visual_odom();
+    AP_VisualOdom *visual_odom = AP::visualodom();
```

Comment:
I think we're using get_singleton() aren't we these days?

Reply:
Not generally.  The access points to singletons is via the `AP::` namespace.  For the most part that's implemented with get_singleton on the relevant class.

Reply:
... also, I believe I've made an even number of grammatical errors in that comment, so I'm all good.